core/mavlink_system.h

```diff
@@ -0,0 +1,223 @@
+#pragma once
+
+#include "global_include.h"
```

Comment:
Just a detail, but following Google guidelines, I like putting the external includes `<...>` first, then the `"..."`. And each group in alphabetical order :-).

Reply:
Sure. I thought fix_style takes care of it.